website/docs/docs/deploy/state-aware-setup.md

```diff
@@ -96,8 +96,8 @@ You can customize with:
 You can either define `loaded_at_field` or `loaded_at_query` but not both.
 :::
 You can also customize with:
-- `updates_on`: Change the default from any to all so it doesn’t build unless all upstreams have fresh data reducing compute even more.
-- `Build_after`: Don’t build a model more often than every x period to reduce build frequency when you need data less often than sources refresh.
+- `build_after`: Fusion-only config enabling you to limit building a model more often than every x period to reduce build frequency when you need data less often than sources refresh.
```

Comment:
switched these because build_after is parent config. Added note about this being a Fusion-only config.